cmd/pdbinspect/main.go

```diff
@@ -1,211 +0,0 @@
-// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
Why is pdbinspect being removed? Is its functionality somewhere else?

Reply:
This seems outdated and will get outdated with more time. Better not to worry about this when refactoring.